src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.OSS.cs

```diff
@@ -17,7 +17,7 @@ public partial class ManifestTest : BaseTest
 			new object[] { "", true, false, },
 			new object[] { "", false, true, },
 			new object[] { "None", true, false, },
-			new object[] { "None", false, false, },
+			new object[] { "None", false, true, },
```

Comment:
...and here...

Reply:
Guess this hits `ManifestTest.DebuggerAttribute()`:

```csharp
if (!isRelease && debugType == "None" && !builder.RunningMSBuild)
    expected = false
```

That still seems odd. Why would the presence/absence of `DebuggerAttribute` depend on xbuild vs. msbuild?